apisix/admin/routes.lua

```diff
@@ -131,6 +133,9 @@ local function check_conf(id, conf, need_id)
         if not ok then
             return nil, {error_msg = err}
         end
+        for name, conf in pairs(conf.plugins) do
```

Comment:
can we do this at lower level? it's not convenient to add this to every admin API,